be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/LaoHandler.scala

```diff
@@ -27,8 +26,7 @@ case object LaoHandler extends MessageHandler {
         socialChannel: Channel = Channel(s"$laoChannel${Channel.SOCIAL_MEDIA_CHIRPS_PREFIX}")
         reactionChannel: Channel = Channel(s"$laoChannel${Channel.REACTIONS_CHANNEL_PREFIX}")
         // we get access to the canonical address of the server
-        config = ServerConf(appConf)
-        address: Option[String] = Some(f"ws://${config.interface}:${config.port}/${config.clientPath}")
+        address: Option[String] = Some(ownClientAddress)
```

Comment:
since this is used in GreetLao, shouldn't it use the external address ?

Reply:
Absoluetely ! Fixed.